src/configs/python.rs

```diff
@@ -44,7 +44,6 @@ impl<'a> Default for PythonConfig<'a> {
                 "tox.ini",
                 "setup.py",
                 "__init__.py",
-                "pixi.toml",
```

Comment:
pixi is a language-agnostic package manager. there are many valid use cases where you have a non-python project but still use pixi. since we now have a proper pixi module, i would be in favor of removing this (as most of the time you should have `.py` files anyway somewhere in your cwd